apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -4,21 +4,21 @@
 import os
 import random
 from copy import deepcopy
+from enum import Enum
+from typing import Any, Dict, TypedDict
```

Comment:
```suggestion
from typing import Any, TypedDict
```
You can use the lowercase `dict` annotation the same way as the lowercase `list`.

Reply:
done, thanks!